src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -157,7 +163,7 @@
 
   <kd-help-section>
     <div layout="row">
-      <md-input-container flex>
+      <md-input-container class="kd-no-margins" flex>
```

Comment:
hmmm...I agree it looks nicer with no margins, but again it is just fiddling. I would rather like to see some consistent concepts. I am just worried that things get messed up as the application ages